nautobot/ui/src/views/Home.js

```diff
@@ -105,18 +105,6 @@ export default function Home() {
                                                     </TagLabel>
                                                 </Tag>
                                             </Tab>
-                                            <Tab>
```

Comment:
Can we just hide these elements instead of removing them?

Reply:
yeah probably should just be `style="display:none"`